crates/store/re_video/src/decode/ffmpeg.rs

```diff
@@ -0,0 +1,730 @@
+                            re_log::warn_once!("Video decode timestamps are expected to monotonically increase unless there was a decoder reset. This is probably a bug in Rerun.");
+                        }
+                        highest_dts = frame_info.decode_timestamp;
+
+                        pending_frame_infos.insert(frame_info.presentation_timestamp, frame_info);
+                    } else {
+                        // There must be an element here, otherwise we wouldn't be in this branch.
+                        #[allow(clippy::unwrap_used)]
+                        break pending_frame_infos.pop_first().unwrap().1;
+                    }
+                };
+
+                let ffmpeg_sidecar::event::OutputVideoFrame {
+                    frame_num: _, // This is made up by ffmpeg sidecar.
+                    pix_fmt,
+                    width,
+                    height,
+                    data,
+                    output_index: _, // This is the stream index. for all we do it's always 0.
+                    timestamp: _, // This is a timestamp made up by ffmpeg_sidecar based on limited information it has.
+                } = frame;
+
+                re_log::trace!(
+                    "{debug_name} received frame: dts {:?} cts {:?} fmt {pix_fmt:?} size {width}x{height}",
+                    frame_info.decode_timestamp,
+                    frame_info.presentation_timestamp
+                );
+
+                debug_assert_eq!(pix_fmt, "rgb24");
+                debug_assert_eq!(width as usize * height as usize * 3, data.len());
+
+                on_output(Ok(super::Frame {
+                    content: super::FrameContent {
+                        data,
+                        width,
+                        height,
+                        format: crate::PixelFormat::Rgb8Unorm,
+                    },
+                    info: super::FrameInfo {
+                        presentation_timestamp: frame_info.presentation_timestamp,
+                        duration: frame_info.duration,
+                        latest_decode_timestamp: Some(frame_info.decode_timestamp),
+                    },
+                }));
+            }
+
+            FfmpegEvent::Done => {
+                // This happens on `pkill ffmpeg`, for instance.
+                re_log::debug!("{debug_name}'s ffmpeg is Done");
+                return;
+            }
+
+            FfmpegEvent::ParsedVersion(ffmpeg_version) => {
+                re_log::debug_once!("ffmpeg version is: {}", ffmpeg_version.version);
+            }
+
+            FfmpegEvent::ParsedConfiguration(ffmpeg_configuration) => {
+                re_log::debug_once!(
+                    "ffmpeg configuration: {:?}",
+                    ffmpeg_configuration.configuration
+                );
+            }
+
+            FfmpegEvent::ParsedDuration(ffmpeg_duration) => {
+                re_log::debug!("ffmpeg duration: {:?}", ffmpeg_duration);
```

Comment:
What is this? What is the unit? Is there a better log message, e.g. that includes the video name?

Reply:
actually let's remove this. ffmpeg doesn't know the duration